### PR TITLE
Task/list updates

### PR DESCRIPTION
**Pull request by charlon**

### CODE REVIEW

File: compass_vue/components/pagination.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        :class="props.theme.list"
+        class="pagination pagination-sm justify-content-end m-0"
+      >
+        <!--<li v-if="props.hasEdgeNav" :class='props.theme.firstPage' @click="props.setFirstPage">
+          <button type="button" v-bind="props.firstPageProps">{{ props.texts.first }}</button>
+        </li>-->
+
+        <!--<li v-if="props.hasChunksNav" :class="props.theme.prevChunk" @click="props.setPrevChunk">
+          <button type="button" v-bind="props.prevChunkProps">{{ props.texts.prevChunk }}</button>
+        </li>-->
```
Comment (nick-falco) [inline compass_vue/components/pagination.vue:15]: I'd remove this instead of commenting

File: compass_vue/components/pagination.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        <!--<li v-if="props.hasChunksNav" :class="props.theme.nextChunk" @click="props.setNextChunk">
+          <button type="button" v-bind="props.nextChunkProps">{{ props.texts.nextChunk }}</button>
+        </li>
+
+        <li v-if="props.hasEdgeNav" :class="props.theme.lastPage" @click="props.setLastPage">
+          <button type="button" v-bind="props.lastPageProps">{{ props.texts.last }}</button>
+        </li>-->
+      </ul>
+      <!--<div v-show="props.hasRecords" :class="props.theme.count">{{ props.count }}</div>-->
```
Comment (nick-falco) [inline compass_vue/components/pagination.vue:46]: same with these

File: compass_vue/main.js [hunk 3]
```diff
@@ -15,8 +15,8 @@ import './css/custom.scss';
 const app = createApp(App);
 
 // custom components
-import Pagination from 'v-pagination-3';
-app.component('paginate', Pagination);
+//import Pagination from 'v-pagination-3';
+//app.component('paginate', Pagination);
```
Comment (nick-falco) [inline compass_vue/main.js:19]: remove these lines too

File: compass_vue/pages/home.vue [hunk 4]
```diff
@@ -8,165 +8,217 @@
...
-                <td>{{item.class_desc}}</td>
-                <td>{{item.major_full_name}}</td>
-                <td>{{item.enrollment_status_desc}}</td>
-                <td>{{item.gender}}</td>
-                <td>{{item.adviser_full_name}}</td>
-              </tr>
-            </tbody>
-          </table>
+        <div class="col-sm-6">
+          <div class="small lh-lg text-end">Displaying: {{ studentsCount }} students</div>
```
Comment (nick-falco) [inline compass_vue/pages/home.vue:62]: This always says displaying 100 students. It should instead say the something along the lines of "Displaying 50-100 of 100 students". It's fixed to the total students count which doesn't seem correct.

Comment (charlon) [reply compass_vue/pages/home.vue:62]: this was originally in the pagination template. but now that display the pagination at the top and bottom, i didn't want to necessarily display the message in both locations. i'll make a prop that controls show/hide of the message

File: compass_vue/components/pagination.vue (hunk 2 above)
Comment (charlon) [reply compass_vue/components/pagination.vue:46]: i didn't want to "lose" some of the original template code while i continue to tweak it. it's easy enough to just grab it from github for reference. i'll remove it.

Comment (nick-falco) [reply compass_vue/components/pagination.vue:46]: Yeah It's generally best practice to not leave commented out code. Just a style thing, no big deal.

File: compass_vue/pages/home.vue (hunk 4 above)
Comment (nick-falco) [reply compass_vue/pages/home.vue:62]: The problem is that the message being shown seems incorrect. It says "Displaying: 100" when there are only 50 per page.

**Pull request merged**